### DISCUSSION TITLE
[Solved] Restore Defaults does not work?

### DISCUSSION THREAD

**Discussion by ETCore7** [Acly/krita-ai-diffusion#1446] [Q&A]
I have been using Live mode. Increasing strength (default 30%) was in big steps since an update, which was very convenient, a few clicks to go to 100%. After trying some other sampler settings (Performance preset) and other models the steps reverted to 5% increase/decrease each click, time consuming. I could not remember the default Sampler Steps and Guidance Strength in the configuration for sure (I assume custom settings is causing the problem), so I clicked 'Restore Defaults'. Unfortunately nothing changes, and the strength steps are still 5% increase/decrease.
![image](https://github.com/user-attachments/assets/58576fe0-fd40-4508-b073-ac415523c025)
![image](https://github.com/user-attachments/assets/d81374d4-c903-42f9-a93a-122c9c924323)
(PS, using 'Realtime - LCM' as sampler causes an error.)
![image](https://github.com/user-attachments/assets/eda28e7d-118b-4fec-b3d5-709515fd1232)

**Comment by Acly** [comment]
Restore Defaults restores the general settings only, it doesn't affect styles. But you can just switch between Sampler presets, and it will set default values?

> (PS, using 'Realtime - LCM' as sampler causes an error.)

Yea not quite sure what it's about, but also no intention to fix. LCM was removed a long time ago in favor of Hyper, it's only displayed if you still happen to use it in one of your styles.

**Comment by ETCore7** [comment]
Thank you. Changing in the configuration window does not revert the settings to default after temporarily switching to another preset, but settings appear to be saved to templates. In the screenshots are the settings after starting a new document from a template, it solved the small increase/decrease steps inconvenience.

![image](https://github.com/user-attachments/assets/f62e5f52-b85b-452f-aeb3-f289cd79e102)

Unfortunately I cannot verify, restarting Krita now says there is an update which throws an error.

![image](https://github.com/user-attachments/assets/0d5418bb-7a79-4b44-b45a-5f5c99f3f1b0)
![image](https://github.com/user-attachments/assets/f30cbb4f-9864-4d0c-81cd-ee468e2d1471)

**Comment by ETCore7** [comment]
Just in case, after waiting for a while the update worked, and the default plugin settings were (indeed) restored through loading an earlier made template. Problem solved.
